File: src/app/value.ts
import { Mask } from './mask';
export class Value {
  ID: number;
  Station_ID: number;
  Type: string;
  DB: number;
  StartByte: number;
  Length: number;
  Val: string;
  Mask_ID: number;
  DBType: number;
  PLC_ID: number;
  Mask: Mask[];
}

File: Host/Host/Model/Value.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Host.Model
{
    public class Value
    {
        public int ID { get; set; }
        public int Station_ID { get; set; }
        public string Type = String.Empty;
        public int DB { get; set; }
        public int StartByte { get; set; }
        public int Length { get; set; }
        public string Val = String.Empty;
        public int Mask_ID = 0;
        public int DBType { get; set; }
        public int PLC_ID { get; set; }
        public List<Mask> Mask = new List<Mask>();
    }
}
